lib/streamlit/hello/Hello.py

```diff
@@ -14,35 +14,24 @@
 
 import streamlit as st
 
+st.write("# Welcome to Streamlit! 👋")
 
-def run():
-    st.set_page_config(
-        page_title="Hello",
-        page_icon="👋",
-    )
```

Comment:
It seems like this no longer runs anywhere (?) is it intentional?

Reply:
Ah. I was thinking originally Hello was the common page, but it's not, so I didn't think we would want this. I'll re-add it back.